HW-2-task1.js

```diff
@@ -0,0 +1,22 @@
+// Implement a function called pinCodeGenerator. This function should take one argument called length and
+// return a randomly generated pin code of a given length.
+
```

Comment:
Hi! The 2nd homework feedback:
1. Good!
2. You do not reassign num variable, so it's better to use const
3. It would be better to use a single regex
4. Good. A better solution is replacing 
if(vovels.includes(letter))
 with 
if(vovels.includes(letter.toLowerCase()))